keps/sig-network/2449-move-externalDNS-out-of-kubernetes-incubator/kep.yaml

```diff
@@ -2,5 +2,8 @@ title: Move ExternalDNS out of Kubernetes incubator
 kep-number: 2449
 authors:
   - "@njuettner"
+approvers:
+  - bowei
+  - thockin
```

Comment:
Got this by following git history through until I found https://github.com/kubernetes/enhancements/pull/1247